rioxarray/raster_array.py

```diff
@@ -456,7 +456,9 @@ def reproject(
         xda.rio.write_coordinate_system(inplace=True)
         return xda
 
-    def reproject_match(self, match_data_array, resampling=Resampling.nearest):
+    def reproject_match(
+        self, match_data_array, resampling=Resampling.nearest, **reproject_kwargs
+    ):
```

Comment:
Mind adding `.. versionadded:: 0.9 reproject_kwargs` ?

Reply:
You got it - I'm not sure I did the comments quite right for the docs.